### ISSUE TITLE
[BUG] [Formatter] broken Flask `{` and `{{` in `<script>` and `<style>`

### ISSUE THREAD

**Issue by Lenormju** [djlint/djLint#2159]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
  * 1.39.5 (released yesterday)
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
  * no open match for "jinja" (closed ones are not relevant)
- [x] I've read the [docs](https://djlint.com)
  * the [profile](https://djlint.com/docs/configuration/#profile) docs does not say much about how it works.

## System Info

- OS: Ubuntu 24.04
- Python version: CPython 3.12.3
- djLint version: 1.39.5
- template language: Jinja

## Issue

When `--format-js` is enabled, Jinja template tags that appear as code inside a `<script>` block are destroyed and turned into invalid JavaScript. The same applies to `--format-css` for `<style>`.

## How To Reproduce

```html
<script>
{% if x %}
a();
{% else %}
b();
{% endif %}
</script>
```

`djlint --reformat --profile jinja --format-js repro.html`

```html
<script>
    {
        %
        if x %
    }
    a();
    {
        %
        else %
    }
    b();
    {
        % endif %
    }
</script>
```

Expected: Jinja tags preserved unchanged, while the surrounding JS get reformatted.

Currently it works on https://djlint.com/demo/ using the above input file, selecting "Jinja" profile, and opting in to "Format JS".

## Notes

* It was already the case in version `1.36.4` (and maybe since the beginning, did not check).
* `{{ ... }}` is preserved only when it sits inside a JS string literal; in any other position it is mangled.
* Current only solution: wrap the block in `{# djlint:off #}` / `{# djlint:on #}`
* My LLM eagerly told me (unverified):
  <details><summary>Possible root cause (unverified)</summary>`formatter/js.py::format_js()` passes the raw `<script>` body to `jsbeautifier.beautify()` with no masking of Jinja tags; `formatter/css.py::format_css()` does the same with `cssbeautifier`. Unlike the HTML pipeline, no placeholder substitution is applied around the beautifier call.</details>
* I can contribute a pull request (human-made) if you want to fix this.

**Comment by Lenormju** [comment]
I confirm that this bug is now fixed in `v1.39.6`, thank you very much for the reactivity ! 😃
